packages/katello/katello/katello.spec

```diff
@@ -183,6 +183,9 @@ Provides a federation of katello services
 * Tue Aug 11 2020 Eric D. Helms <ericdhelms@gmail.com> - 3.18.0-0.1.master
 - Bump to 3.18.0
 
+* Tue Aug 11 2020 Jeremy Lenz <jlenz@redhat.com> - 3.17.0-0.2.master
```

Comment:
Bump this to `3.18.0-0.2.master` and move this entry to just below `%changelog` and you'll be good to go.